### DISCUSSION TITLE
🚜 Farming Equipment & Buildings

### DISCUSSION THREAD

**Discussion by adamhannigan** [sunflower-land/sunflower-land#2148] [Ideas]
This proposal outlines the introduction of new farming equipment and buildings that will enhance the player's experience and make the game more engaging.

**Goal**

Our goal for Q1 of 2023 is to improve the gameplay for new players and make the game more engaging for existing players.

The aim is to encourage new players, create a stronger community and overall a flourishing ecosystem.

**Problem**

Most in-game items are inaccessible to new players since they have already been sold out or are too expensive when they become available. This provides limited milestones and goals for new players when they enter the game - From a Game Design perspective, the difficulty curve is unbalanced.

We need to foster an environment where new players do not feel disadvantaged and have a long-list of items that they can attain if they keep playing the game.

**Excludes**

This proposal focusses on the addition of new items to increase engagement. Other planned changes are not included in this discussion:

- Lottery
- Improve Expansion Requirements / Re-rolls
- Moving resources (Landscape Shovel)
- Multiple Bumpkins
- Enhanced skills / skill resets

## Helios Farming Equipment

The following farming equipment will become available at the Helios Blacksmith with unlimited supply. Since these items are always available for purchase in-game, they cannot be withdrawn.

We will introduce an Area of Impact Mechanic on these items.

**Sprinkler**

Adjacent Crops produce +0.25.

Level: 3, 8, 14, 26
Cost: TBD
Dimensions: 1x1
Limit: 4
Area of Impact: 3x3
Notes: 1 Sprinkler can affect at most 8 crops

<img width="252" alt="Screen Shot 2023-02-02 at 8 07 39 am" src="https://user-images.githubusercontent.com/11745561/216163538-5c573d04-1a9d-4ad0-987f-d25b2d3a619d.png">

**Scarecrow 1**

Basic Crops grow 20% faster (Sunflowers, Potatoes and Pumpkins)

Level: 3
Cost: TBD
Dimensions: 1x1
Notes: Name/art TBD
Area of Impact: 6x6

**Scarecrow 2**

Advanced Crops grow 20% faster (Carrot, Cabbage, Beetroot, Cauliflower)

Level: 6
Cost: TBD
Dimensions: 1x1
Notes: Name/art TBD
Area of Impact: 6x6

**Scarecrow 3**

Expert Crops grow 20% faster (Radish, Parsnip, Wheat, Kale)

Level: 16
Cost: TBD
Dimensions: 1x1
Notes: Name/art TBD
Area of Impact: 6x6

**Bale**

Adjacent Chickens Produce +0.2 Eggs

Level: 10, 30
Cost: TBD
Dimensions: 2x2
Supply: 2
Notes: 1 Bale can accommodate 10 chickens
Area of Impact: 4x4

<img width="182" alt="Screen Shot 2023-02-02 at 8 08 17 am" src="https://user-images.githubusercontent.com/11745561/216163619-b61bd99b-aa96-430a-be62-0394cd0cbbab.png">

**Tractor**

Adjacent Fruit Grows 25% faster. A tractor moves vertically up and down.

Level: 14
Cost: TBD
Dimensions: 2 x 4
Supply: 1
Notes: 1 Tractor can accommodate 6 trees
Area of Impact: 3x5

**Wheel Barrow**

Collect bonus +0.2 Fruit

Level: 26
Cost: TBD
Dimensions: 2x2
Supply: 1
Area of Impact: 6x6

<img width="154" alt="Screen Shot 2023-02-02 at 8 08 34 am" src="https://user-images.githubusercontent.com/11745561/216163656-837a4843-ef1f-43ed-8b7f-a516550b204d.png">

**Pets**

_Separate Proposal: _ Feed pets and get farm boosts

## Buildings

The following buildings will become available for players to craft and enhance their farming abilities

**Silo**

Increase seed stock by 20%

Level: 9
Cost: TBD
Dimensions: 3x2

**Toolshed**

Increase tool stock by 50%

Level: 12
Cost: TBD
Dimensions: 3x2

**Lumberjack**

Bonus +0.2 Wood from trees

Level: 8
Cost: TBD
Dimensions: 3x2

**Quarry**

Bonus +0.2 Stone from stone rocks. +0.1 Iron from iron rocks

Level: 12
Cost: TBD
Dimensions: 3x2

**Composter**

Bonus +0.1 Crops

Level: 18
Cost: TBD
Dimensions: 3x2

<img width="175" alt="Screen Shot 2023-02-02 at 8 09 29 am" src="https://user-images.githubusercontent.com/11745561/216163803-00efd636-9bd0-43c1-8270-565e00643400.png">

### Game Balance

To accomodate for the increase in boosts, the prices of buildings & expansion requirements will be adjusted assuming that players now have an increased crop & resource yield.

Prices are currently TBD as they will rely on final buffs of each item.

**Comment by Telknub** [comment]
This is really cool. Loving all this items. Can't wait for them!

**Comment by IvanSkazkin** [comment]
Will these new items work with the old ones? For example, the scarecrow t1-t3 from the old version with the scarecrow t1-t3 of the new version?

**Reply by adamhannigan** [reply]
Yes, buffs will be applied for both

**Reply by IvanSkazkin** [reply]
> Да, баффы будут применены к обоим

Super!

**Comment by brian-man** [comment]
> Lumberjack
> Bonus +0.2 Wood from trees

Can this be a little dude rather than a building?

**Reply by adamhannigan** [reply]
Ideally we want something that takes up more space - as players expand, we need items that take up space to incentivise them to keep expanding 👍

**Reply by rasheed19871** [reply]
that will be great

**Reply by TakeMyMuffin** [reply]
Great stuff above, but I was wondering why the tractor was sized so big.  Thought scale seemed a little odd with wheel barrel being same size as a tree and 4x that of bumpkin.  Also seems like wheelbarrow should come before tractor.

**Comment by future2back** [comment]
I'm not in favor of re-rolls. Any kind of change that takes the game to be a **pay to win** I disagree and this looks like it.
If today the scenario is already delicate, imagine with this change? I believe it would be unsustainable. 
It's easier to buy the resources than to make it on your farm, it doesn't make sense. As long as the balances still affects the T3 and green amulet, it's going to be very difficult.

**Reply by future2back** [reply]
There is a giant chasm between the small and the big. This needs to change urgently. The big ones are not afraid to play aggressively because they profit more and more from it while small ones play aggressively until they reach the pay to pay point, that is expensive. How much time and money a player that recently minted an account without NFTs have to invest to reach lvl 40 and full expansions? Can you tell me how I could invite a friend to play? What are we offering to new players? I'm talking about entertainment, players need to spend time on something other than creating new resources only.

**Reply by adamhannigan** [reply]
Can you please clarify what part of the proposal you are referring to? This discussion excludes `re-rolls` and land expansion requirements. @future2back 

The proposal may not of been clear enough - the purpose is to accomodate new players.

There is no max supply on these items. They are low cost and targeted for new players to craft and proceed in the game.

**Reply by future2back** [reply]
Sorry bro, it was my mistake.

**Reply by darcyd23** [reply]
@scooter2023

**Reply by Scooter2023** [reply]
@lilyg23

**Comment by IvanSkazkin** [comment]
It is possible to add nft +5% to food. It is difficult for beginners to level up.

**Comment by SuperSynapse** [comment]
I think this is great for all players!

Will these be proper NFTs and tradeable like chickens? or is the intent for them to be locked to our farm?

Additionally the only issue I have is making 2 types of 3 tiered scarecrows. I think this will just confuse new players and make a new conversation caveat. Is it possible to make the "new scarecrows" something that isn't a scarecrow?

**Reply by brian-man** [reply]
> Will these be proper NFTs and tradeable like chickens? or is the intent for them to be locked to our farm?

Covered explicitly:
> The following farming equipment will become available at the Helios Blacksmith with unlimited supply. Since these items are always available for purchase in-game, they cannot be withdrawn.

**Reply by ShiningMinions** [reply]
> Will these be proper NFTs and tradeable like chickens? or is the intent for them to be locked to our farm?

I personally love this mechanic. I love how there's a section of the game where I'm not competing against other people to acquire items. It gives a good single-player experience to SFL and also fantastic for new players IMO.

> Additionally the only issue I have is making 2 types of 3 tiered scarecrows. I think this will just confuse new players and make a new conversation caveat. Is it possible to make the "new scarecrows" something that isn't a scarecrow?

I agree with you. It could get confusing between the different sets of scarecrows.

**Reply by Aeon-LlamaMod** [reply]
I believe these would be similar to the immortal pear and not be tradeable. More or less in line with them improving your farm but not withdrawable.

**Reply by alphacruze** [reply]
I do not think the scarecrows are "tiered" per say, they completely different scarecrows, different looks different buffs, only thing needed is the proper level for said scarecrow, so its less of a "tier" well in the sense of how SFL has done tiered NFT's

**Comment by CaptainGokuu** [comment]
These additions reduce the rarity and value of Nfts. Old players have spent their times and put a lot of effort into making these nfts. These changes, which can be made to attract new players and help them, are stealing the old players' labor and even their money if we think that their current nft value will decrease. For example, why would anyone buy "Wood Nymph Wendy" after this time to get +0.2 wood in-game? I accept that the game is difficult for new players, but if you want to help these players, you need to do something about resources that do not already have a boost nfts.

**Reply by Telknub** [reply]
It doesnt tho. Just because they released wendy, the beaver prices did not go down. Also these items are not withdrawable, so players who want more buffs would still need to mint/buy other SFTs.

Saying that these items will reduce value and rarity of other SFTs makes no sense. By your logic then we cannot have any new SFT on the game that is related to an old feature....

**Reply by IvanSkazkin** [reply]
It's the other way around. The more you expand, the more resources you need. Therefore, the need does not decrease vice versa. Also, these new nfts work with the old ones. The old guys get a lot more boost than the newbies.

**Reply by alphacruze** [reply]
these are like buildings, non-withdrawable, non tradable and meant to provide a manner for progression for new players

As, the way the game is right now, new players are unlikely to pick up the game and continue playing, why? because its too damn brutal for newer players. Which is also why all the crop buffs and even the chicken one has a "range" 

All of these things are meant to enhance the GAMING experience, while also not messing too much with the outside NFT market

**Comment by TheDcol** [comment]
Instead of scarecrow I II and III, have you thought about naming them something else as there are already scarecrows (nancy, scarecrow and kuebiko) in the game?

An idea would be that instead of scarecrows, they would be animated 'farmers' with a water tank or rake/shovels or something. You can call them:

- Farmer Adam
- Farmer Craig
- Farmer Romy/Spencer/Harnoor etc.

**Reply by adamhannigan** [reply]
Hahah, Farmer Dcol.

Yes, these names are placeholders right now - open to any suggestions 🤗

**Reply by paduv** [reply]
Why not Owls ?
Owls eat pests, that should make for better crops, no?
& we don't have owls yet in the game i believe, a whole new ground to cover :)

Also, 'Owls are not what they seem'... (suspenseful hum)

**Reply by TheDcol** [reply]
> Hahah, Farmer Dcol.
> 
> Yes, these names are placeholders right now - open to any suggestions 🤗

I only grow basil and tomatoes ahah

**Reply by adamhannigan** [reply]
@paduv Owls is a great suggestion 😮 This would be great to add as an item.

It may just be harder to create 3 variations of an owl. 1 or 2 may suffice 👌

**Reply by blastaman** [reply]
We have the three tiers of scarecrows, how about three tiers of Tin-Men next?

T1 - rusty tin man 
T2 - shiny tin man
T3 - Iron Man

**Reply by blastaman** [reply]
Then one day we can get the cowardly lion!

**Reply by SopNami** [reply]
I don't like the name scarecrow it's better another name like this reply

**Reply by alphacruze** [reply]
Why name them according to what they are, just give them cute names. I mean we have Nancy for T1 scarecrow, and Kuebiko for T3

So, we might as well have a naming competition on Twitter, where people can suggest names and then put it to a vote

**Comment by alphacruze** [comment]
This is great

This is what we need to bring in new players, glad that the team sees the problem and is working on addressing it

However,a slight suggestion, perhaps keeping the cost of the first 2 expansions and cost of the buildings below level 10 and so on the same

To facilitate that ability for new gamers to progress through those in a relatively short period of time.
Which would likely still be a month for new players unless they want to waste a whole lot of seeds... which they definitely cannot afford to do

**Reply by adamhannigan** [reply]
Nice suggestion @alphacruze 👏 

The team will be addressing the progression curve for expansions over the coming weeks. We aim to make it a lot more engaging to hook the new players 😄

**Reply by alphacruze** [reply]
I look forward to news on this

**Comment by diasfigor** [comment]
Very good. I couldn't expect less from you. These new features will do a lot of good to our beloved game. Best team. **SunFlower Forever!!!**

**Comment by Philifuture11** [comment]
With these new equipments and buildings, it will be necessary to be able to move crops, trees, etc. Should we include the moving discussion here too?

**Reply by adamhannigan** [reply]
We've just created the proposal for that 😄 

https://github.com/sunflower-land/sunflower-land/discussions/2195

**Comment by Gobira54** [comment]
The ideal thing would be to have an event pass with points in daily missions, and a way to progress.

For example, a pass lasting 60 days, the tasks would give 10 points per day, but each "Level" of the path would be 20 points.

The rewards of this path could be temporary Buffs of 1, 3 and 7 days, for example: +0.1 stone, +0.25 in croop, +0.15 in wood.

It could also have tickets/pass currency, which would give access to a specific store where you could buy items that would give permanent Buffs, and that these items could be Upgraded, joining two of the same level + special items + SFL.

In the way of the pass, it could also, at a certain level of the way, give the right to the person to mint a certain NFT if they have the required resources for the mint, this would replace the auctioneer.

Daily quests could be simple things, for example:
- Dig 2x on treasure island

- Cut down 5x trees

- Spoon 10 croop

- Consume 50 xp

It could vary by day.

**Comment by rasheed19871** [comment]


**Reply by LucasAG95** [reply]
@adamhannigan spam

**Comment by brian-man** [comment]
> Bale
> Adjacent Chickens Produce +0.2 Eggs
> Notes: 1 Bale can accommodate 10 chickens
> Level: 10, 30

Any thoughts on adding a 3rd Bale at L50?  People with 30 chickens would love to know. :)

**Comment by brian-man** [comment]
> Tractor
> Adjacent Fruit Grows 25% faster. A tractor moves vertically up and down.
> Notes: 1 Tractor can accommodate 6 trees

Players at max expansions already have 7 trees.  Please consider making this boost something other than time-related.  Having all trees after the 6th on a different timer/schedule will be annoying.

**Comment by brian-man** [comment]
> Silo
> Increase seed stock by 20%

> Toolshed
> Increase tool stock by 50%

Please revisit the base supply for all seeds and tools to make sure that these 20% and 50% values result in nice round numbers.  Fractional values for supply will be annoying.

**Comment by DeshFarm** [comment]
Muito boas as ideias...acho que irá ainda mais auxiliar as pessoas que estão ingressando no jogo agora, e ainda vai incentivar a galera que já joga a jogar com mais engajamento ainda mais...e ainda acho muito bacana essa participação da comunidade na coleta de informações acho que sempre a tendência é crescer mais e mais com essa metodologia.

Very good ideas... I think it will help people who are joining the game even more now, and will even encourage people who already play to play with even more engagement... and I still think this community participation is very cool in collecting information, I think the tendency is always to grow more and more with this methodology.

**Comment by SopNami** [comment]
I think that since they are not, they can be acquired indefinitely, they must have a useful time for it to be completely damaged or that resources have to be spent to repair it.

**Reply by alphacruze** [reply]
Well with that kind of logic, everything that gives a boost to the game should be "damageable" and need to be repaired after a certain amount of time in usage

**Comment by alphacruze** [comment]
Honestly this is still my favorite proposal, a method to add boosts that any can acquire at any time, especially helpful to new players

These kinds of things make it feel like it's a game, and not a cash grab where the first to acquire something can make a profit.

**Comment by brian-man** [comment]
For all 3 scarecrows:
> Dimensions: 1x1
> Area of Impact: 6x6

Where is the scarecrow positioned in the 6x6 area grid?

**Reply by adamhannigan** [reply]
Thanks for pointing that out, the sizing won't work evenly. Initially the dimensions were 2x2.

We will need to adjust this 👌

**Comment by brian-man** [comment]
For Tractor, the Area of Impact value doesn't make sense to me.
> Dimensions: 2 x 4
> Notes: 1 Tractor can accommodate 6 trees
> Area of Impact: 3x5

3x15 is 15.  6 trees occupy 24 space.

**Reply by Nimr1** [reply]
Hi, Believe due to Tractor moving up and down  the Area of impact also moves with it. so  Theoretically if it moves up and down on  10 plot line he will cover ample Fruits   in various positions

**Comment by SarielWang93** [comment]


**Reply by brian-man** [reply]
Unrelated to the topics at hand.  Why post in the wrong place?  At best it's rude.

**Reply by SarielWang93** [reply]
Thanks, I will find another place to give feedback, delete it here

**Comment by SarielWang93** [comment]
Can Toolshed increase the number of sand shovels and sand drills in stock？

**Reply by NKbS4s38V** [reply]
> Can Toolshed increase the number of sand shovels and sand drills in stock？

 i dont think it is something that the new players need

**Comment by NKbS4s38V** [comment]
Agree, nice ideas that will encourage and help new players.👍👍👍👍👍

**Comment by andrelopes2001** [comment]
Will the plots/resources have a limit of bonuses? Or can a plot have all bonuses listed if the sft's are arranged properly?
